src/docker-compose-testkit.ts

```diff
@@ -67,6 +68,9 @@ export interface Compose {
 }
 
 export function compose(pathToCompose: string, options?: ComposeOptions): Compose {
+  if (!existsSync(pathToCompose)) {
+    throw new Error(`docker-compose file not found at ${pathToCompose}`)
+  }
```

Comment:
not really related to the PR.
just a simple "early abort".
can remove if you want.